source/2019-08-15-octane-release-plan.md

```diff
@@ -0,0 +1,249 @@
+
+### application-template-wrapper
+
+In classic Ember, your entire application is automatically wrapped in a `<div>` with the class `ember-application`. Octane applications do not automatically insert this uneccessary `<div>`.
+
+If your CSS (or JavaScript) relies on this `<div>`, you can explicitly add it to your `application.hbs` or refactor your code to no longer rely on it.
+
+You opt in to this change by
+
+    $ ember feature:disable application-template-wrapper
+
+### jquery-integration
+
+In classic Ember, jQuery is automatically included, and `this.$` inside of component classes uses jQuery to select from the DOM. Ember Octane does not include jQuery automatically, nor does it include any direct jQuery integration in the component API.
+
+Ember's internals no longer depend on jQuery. If your app still uses jQuery directly and you want to continue using it, you can add `@ember/jquery` to your `package.json` and disable this classic feature. You will still be able to use all the other features of Octane.
+
+Alternatively, you could migrate away from using jQuery in your application, but it is not necessary. Octane's other features work just fine either way.
+
+You opt in to this change by
+
+```sh
+$ ember feature:disable jquery-integration
+```
+
+If you still want to use jQuery in your application, make sure to add `@ember/jquery` to your `package.json`.
+
+### template-only-glimmer-components
+
+In classic Ember, components that have a template but no JavaScript file get assigned a default component (`EmberComponent.extend()`). This means that you can refer to properties on this implicit component using two-way bindings, and those properties will be implicitly created for you.
+
+Octane does not implicitly create a component class for you, and the `this` context inside the template is `undefined`. You can still refer to passed-in arguments using `@name` and yield to blocks using `yield`, but references to properties on the component like `{{someProp}}` or `{{this.someProp}}` will not work.
+
+Enabling `template-only-glimmer-components` will opt you in to the new behavior. The easiest way to migrate existing applications to this flag is to create an explicit JavaScript file for each template-only component that refers to properties on the implicit component.
+
+Alternatively, you could eliminate any references to the implicit component by renaming arguments from `{{someArg}}` to `{{@someArg}}` and removing any reliance on two-way bindings on the implicit component.
+
+When done, you can opt in to the new behavior:
+
+```sh
+$ ember feature:enable template-only-glimmer-component
+```
```

Comment:
Is there a migration guide or blog post for this optional feature we can link people to?

Reply:
Not that I know of.